lib/styleguide/app/initializers/styleguide-routing.js

```diff
@@ -0,0 +1,13 @@
+// my-app/lib/styleguide/app/initializers/styleguide-routing.js
+
+import MyAppRouter from 'code-corps-ember/router';
+import styleguideRoutes from '../styleguide-routes';
+
+export function initialize() {
+  MyAppRouter.map(styleguideRoutes);
+}
+
+export default {
+  name: 'styleguide-routes',
+  initialize
+};
```

Comment:
Probably should change the wording from `MyApp` to `CodeCorpsEmber`.

Also the comment on line 1